### DISCUSSION TITLE
personal-summary/cse/锐评海南大学网络空间安全学院（密码学院）和密码学实验班-匿名

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#15] [Announcements]
# /personal-summary/cse/%E9%94%90%E8%AF%84%E6%B5%B7%E5%8D%97%E5%A4%A7%E5%AD%A6%E7%BD%91%E7%BB%9C%E7%A9%BA%E9%97%B4%E5%AE%89%E5%85%A8%E5%AD%A6%E9%99%A2%EF%BC%88%E5%AF%86%E7%A0%81%E5%AD%A6%E9%99%A2%EF%BC%89%E5%92%8C%E5%AF%86%E7%A0%81%E5%AD%A6%E5%AE%9E%E9%AA%8C%E7%8F%AD-%E5%8C%BF%E5%90%8D

A magical documentation generator.

https://hainanu-application.github.io/#/personal-summary/cse/%E9%94%90%E8%AF%84%E6%B5%B7%E5%8D%97%E5%A4%A7%E5%AD%A6%E7%BD%91%E7%BB%9C%E7%A9%BA%E9%97%B4%E5%AE%89%E5%85%A8%E5%AD%A6%E9%99%A2%EF%BC%88%E5%AF%86%E7%A0%81%E5%AD%A6%E9%99%A2%EF%BC%89%E5%92%8C%E5%AF%86%E7%A0%81%E5%AD%A6%E5%AE%9E%E9%AA%8C%E7%8F%AD-%E5%8C%BF%E5%90%8D

**Comment by chenzongyao200127** [comment]
绷不住了🚀

**Reply by tonglingxiangyu** [reply]
海大网安人集合！

**Reply by beatbox505** [reply]
正确的，一针见血的，客观的，不会了

**Comment by Plucky923** [comment]
难绷的

**Comment by Fontzs** [comment]
治好了我的低血压👍
